src/test/java/com/cdancy/artifactory/rest/features/ArtifactApiMockTest.java

```diff
@@ -108,6 +108,26 @@ public void testCopyArtifact() throws Exception {
         }
     }
 
+    public void testMoveArtifact() throws Exception {
+        MockWebServer server = mockArtifactoryJavaWebServer();
+
+        String payload = payloadFromResource("/artifact-copy.json");
+        server.enqueue(new MockResponse().setBody(payload).setResponseCode(200));
+        ArtifactoryApi jcloudsApi = api(server.getUrl("/"));
+        ArtifactApi api = jcloudsApi.artifactApi();
+        try {
+            RequestStatus requestStatus = api.moveArtifact("libs-snapshot-local", "hello/world", "ext-snapshot-local/hello/world");
+            assertNotNull(requestStatus);
+            assertTrue(requestStatus.errors().size() == 0);
+            assertTrue(requestStatus.messages().size() == 1);
+            assertTrue(requestStatus.messages().get(0).level().equalsIgnoreCase("info"));
+            assertSent(server, "POST", "/api/move/libs-snapshot-local/hello/world?failFast=1&suppressLayouts=0&to=ext-snapshot-local/hello/world", MediaType.APPLICATION_JSON);
+        } finally {
+            jcloudsApi.close();
+            server.shutdown();
+        }
+    }
```

Comment:
@kalap92 can you also write a live test for the negative case? Meaning try to request to move an artifact that doesn't exist and assert the proper outputs/response/etc.

Reply:
Same for live tests as well.